src/hooks/useProjects.ts

```
@@ -406,382 +412,273 @@ export const useProjects = () => {
-        `)
-        .eq('project_id', projectId)
-        .eq('user_id', currentUser.id)
-        .order('created_at', { ascending: true });
+        .select('status')
+        .eq('project_id', projectId);
 
       if (tasksError) throw tasksError;
 
-      // Transform Supabase data to match our Project interface
-      const transformedProject: Project = {
+      const completedTasks = tasksData?.filter((task: any) => task.status === 'DONE').length || 0;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)